app/src/main/res/values/strings.xml

```diff
@@ -348,7 +348,7 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="reading">Reading</string>
     <string name="read">Read</string>
     <string name="table">Table</string>
-    <string name="how_much_columns_press_table_button_long_to_start_table">How much columns? Press table button long to start a table (insert header), press short to add a row.</string>
+    <string name="how_many_columns_press_table_button_long_to_start_table">How many columns? Press table button long to start a table (insert header), press short to add a row.</string>
```

Comment:
Please keep the string id and change only the english string. Translations are completely managed by translation service.

Reply:
`how_much_columns_press_table_button_long_to_start_table` is technically also not totally correct, and it seems strange to use how_much for the string id, and then "how many" for the real text no ?

Reply:
"Many" should be used instead of "much" because the columns can be counted (2,3,4,...).

Reply:
As said, please update the english strings, as required. Keep the IDs however, they dont matter for this.

Reply:
Ok, done !